docs/howto/garbage-collection.md

```diff
@@ -449,20 +449,61 @@ The uncommitted GC will not clean:
 1. lakeFS server version must be at least [v0.87.0](https://github.com/treeverse/lakeFS/releases/tag/v0.87.0). 
 If your version is lower, you should first upgrade.
 2. Read the [limitations](#limitations) section.
+3. Use [rclone](https://rclone.org/) for backup and restore
+
+
+Running uncommitted GC performed by the following steps:
+
+1. Mark the files to deleted - summary and report will be generated under '<repository storage namespace>/_lakefs/retention/gc/uncommitted/<mark id>/'.
+   The mark id sortable time based ID, each run output into a new one.
+
+   ```bash
+   spark-submit \
+       --conf spark.hadoop.lakefs.api.url=<LAKEFS_ENDPOINT> \
+       --conf spark.hadoop.fs.s3a.access.key=<AWS_ACCESS_KEY_ID> \
+       --conf spark.hadoop.fs.s3a.secret.key=<AWS_SECRET_ACCESS_KEY> \
+       --conf spark.hadoop.lakefs.api.access_key=<LAKEFS_ACCESS_KEY_ID> \
+       --conf spark.hadoop.lakefs.api.secret_key=<LAKEFS__SECRET_ACCESS_KEY> \
+       --class io.treeverse.gc.UncommittedGarbageCollector \
+       --packages org.apache.hadoop:hadoop-aws:2.7.7 \
+       lakefs-spark-client-312-hadoop3-assembly-0.6.0.jar \
+       <REPO_NAME> <REPO>
+   ```
+
+2. Backup (recommended) - can be done after examine the above content. Backup will copy the objects marked to be deleted for run ID to a specified location.
+   Follow [rclone documentation](https://rclone.org/docs/) to configure remote access to lakeFS storage and the backup location.
+
+   ```shell
+   rclone --include "*.txt" cat "remote:<lakefs storage namespace>/_lakefs/retention/gc/uncommitted/<mark id>/deleted.text/" | \
+     rclone -P --no-traverse --files-from - copy <lakefs storage namespace> <backup storage location>
+   ```
+
+4. Sweep - delete reported objects to delete based on mark ID
+
+   ```bash
+   spark-submit \
+       --conf spark.hadoop.lakefs.gc.mark_id=true \
+       --conf spark.hadoop.lakefs.gc.do_sweep=true \
+       --conf spark.hadoop.lakefs.api.url=<LAKEFS_ENDPOINT> \
+       --conf spark.hadoop.fs.s3a.access.key=<AWS_ACCESS_KEY_ID> \
+       --conf spark.hadoop.fs.s3a.secret.key=<AWS_SECRET_ACCESS_KEY> \
+       --conf spark.hadoop.lakefs.api.access_key=<LAKEFS_ACCESS_KEY_ID> \
+       --conf spark.hadoop.lakefs.api.secret_key=<LAKEFS__SECRET_ACCESS_KEY> \
+       --class io.treeverse.gc.UncommittedGarbageCollector \
+       --packages org.apache.hadoop:hadoop-aws:2.7.7 \
+       lakefs-spark-client-312-hadoop3-assembly-0.6.0.jar \
+       <REPO_NAME> <REGION>
+   ```
+
+   In case we would like to always run the uncommitted GC without backup, passing `--conf spark.hadoop.lakefs.gc.do_sweep=true` to theprevious command will perform both steps.
```

Comment:
```suggestion
   In case we would like to always run the uncommitted GC without backup, passing `--conf spark.hadoop.lakefs.gc.do_sweep=true` to the previous command will perform both steps.
```